openshift/4.0/README.md

```diff
@@ -225,455 +225,30 @@ oc project 354028-dev
 oc process -f pims-db-storage.yaml --param-file=dev.env | oc create --save-config=true -f -
 ```
 
-### Configure Database
+### API Database
```

Comment:
This readme was getting too long.  Moved sections into their own README files and put them closer to their related templates.